ext/scheduler/airflow/dag/template/dag.2.1.py.tmpl

```diff
@@ -189,7 +189,7 @@ init_container_{{$hookName}} = k8s.V1Container(
     image_pull_policy=IMAGE_PULL_POLICY,
     env=init_env_vars + [
         k8s.V1EnvVar(name="INSTANCE_TYPE", value='{{$.ExecutorHook}}'),
-        k8s.V1EnvVar(name="INSTANCE_NAME", value='{{$hookName}}'),
```

Comment:
have we ensure that this change won't break any related existing behavior mba @arinda-arif ?